### PR TITLE
Updates for Flutter 3

### PR DESCRIPTION
**Pull request by johnpryan**
Run "dart fix --apply" on the Flutter beta channel and bump the patch version of this package.

```
dart fix --dry-run  
Computing fixes in cached_network_image (dry run)... 26.9s

30 proposed fixes in 8 files.

example/lib/main.dart
  unnecessary_import • 1 fix

lib/src/cached_image_widget.dart
  unnecessary_import • 1 fix

lib/src/image_provider/cached_network_image_provider.dart
  invalid_null_aware_operator • 2 fixes

lib/src/image_provider/multi_image_stream_completer.dart
  invalid_null_aware_operator • 1 fix

test/image_cache_manager_test.dart
  invalid_null_aware_operator • 4 fixes

test/image_provider_test.dart
  invalid_null_aware_operator • 10 fixes
  unnecessary_import • 1 fix

test/image_stream_completer_test.dart
  unnecessary_non_null_assertion • 6 fixes

test/image_widget_test.dart
  invalid_null_aware_operator • 4 fixes
```

### CODE REVIEW

**Review by Nanow — APPROVED** [review]

File: cached_network_image/pubspec.yaml [hunk 1]
```diff
@@ -21,5 +21,4 @@ dev_dependencies:
   file: ^6.1.0
 
 environment:
-  flutter: ">=1.24.0-10.2.pre"
-  sdk: ">=2.12.0 <3.0.0"
+  sdk: ">=2.17.0-0 <3.0.0"
```
Comment (renefloor) [inline cached_network_image/pubspec.yaml:24]: I guess we can keep flutter here
  flutter: ">=3.0.0"

File: cached_network_image/pubspec.yaml [hunk 2]
```diff
@@ -2,7 +2,7 @@ name: cached_network_image
 description: Flutter library to load and cache network images.
   Can also be used with placeholder and error widgets.
 homepage: https://github.com/Baseflow/flutter_cached_network_image
-version: 3.2.0
+version: 3.2.1
```
Comment (orestesgaolin) [inline cached_network_image/pubspec.yaml:5]: Are we sure that it's just a minor change? Maybe we should bump it by 0.1.0?

Comment (johnpryan) [reply cached_network_image/pubspec.yaml:5]: I wasn't sure either, but since the API of this package isn't changing I think this is the most correct 🤔

**Comment by viralkachhadiya** [comment]
#740

**Comment by lucasrueda** [comment]
Excellent work. Thanks

**Pull request merged**